Configurations/10-main.conf

```diff
@@ -562,7 +562,7 @@ my %targets = (
         cppflags         => add(threads("-D_REENTRANT")),
         ex_libs          => add("-ldl", threads("-lpthread")),
         bn_ops           => "SIXTY_FOUR_BIT",
-        asm_arch         => 'ia64',
+        asm_arch         => 'ia64_hpux32',
```

Comment:
I would call the arch ia64_32 as I strongly suspect this is not hpux specific and any other ia64 32 bit platform (if there are any) would suffer from it too.